awx/ui/test/e2e/tests/test-users-crud.js

```diff
@@ -0,0 +1,54 @@
+/* Tests for the user CRUD operations. */
+import { genID } from '../utils';
+
+const testID = genID();
+
+const store = {
+    organization: {
+        name: `org-${testID}`
+    },
+    user: {
+        username: `user-${testID}`,
+        email: `email-${testID}@example.com`,
+        password: `${testID}`,
+    },
+};
+
+module.exports = {
+    before: client => {
+        client.login();
+        client.waitForAngular();
+
+        client.inject(
+            [store, 'OrganizationModel'],
+            (_store_, Model) => new Model().http.post({ data: _store_.organization }),
+            ({ data }) => { store.organization = data; }
+        );
+    },
+    'create an user': client => {
+        const users = client.page.users();
+        users.load();
+        users.section.list
+            .waitForElementVisible('@add')
+            .click('@add');
+        users.section.add
+            .waitForElementVisible('@title')
+            .setValue('@organization', store.organization.name)
+            .setValue('@email', store.user.email)
+            .setValue('@username', store.user.username)
+            .setValue('@password', store.user.password)
+            .setValue('@confirmPassword', store.user.password)
+            .click('@save');
+        client.waitForSpinny();
+        users.section.list.section.search
+            .setValue('@input', store.user.username)
+            .click('@searchButton');
+        client.waitForSpinny();
+        const row = '#users_table .List-tableRow';
+        users.waitForElementNotPresent(`${row}:nth-of-type(2)`);
+        users.expect.element(row).text.contain(store.user.username);
```

Comment:
Another good thing to test here is that the badge count updates. In this case, we'd expect the text content of the badge to show exactly '1'